fhir-search/src/test/java/com/ibm/fhir/search/test/InclusionParameterParseTest.java

```diff
@@ -328,6 +348,34 @@ public void testIncludeValidTargetType() throws Exception {
         assertTrue(selfUri.contains(queryString));
     }
 
+    @Test
+    public void testRevIncludeWithSort() throws Exception {
+        Map<String, List<String>> queryParameters = new HashMap<>();
+        Class<Organization> resourceType = Organization.class;
+        String queryString = "&_revinclude=Patient:general-practitioner:Organization&_sort=name";
+        String sortParmCode = "name";
+        InclusionParameter revIncParm = new InclusionParameter("Patient", "general-practitioner", "Organization", null, true);
+
+        queryParameters.put("_sort", Collections.singletonList("name"));
+        queryParameters.put("_revinclude", Collections.singletonList("Patient:general-practitioner:Organization"));
+        FHIRSearchContext searchContext = SearchUtil.parseQueryParameters(resourceType, queryParameters);
+        assertNotNull(searchContext);
+
+        assertNotNull(searchContext.getSortParameters());
+        assertEquals(searchContext.getSortParameters().size(), 1);
+        SortParameter sortParm = searchContext.getSortParameters().get(0);
+        assertEquals(sortParm.getCode(), sortParmCode);
+        assertEquals(sortParm.getDirection(), Sort.Direction.INCREASING);
+        assertEquals(sortParm.getType(), Type.STRING);
+        assertFalse(searchContext.hasIncludeParameters());
+        assertTrue(searchContext.hasRevIncludeParameters());
+        assertEquals(searchContext.getRevIncludeParameters().size(), 1);
+        assertEquals(searchContext.getRevIncludeParameters().get(0), revIncParm);
+
+        String selfUri = SearchUtil.buildSearchSelfUri("http://example.com/Patient", searchContext);
+        assertTrue(selfUri.contains(queryString));
+    }
+
```

Comment:
might want to add a test for _include/_revinclude with _summary=text

Reply:
Added.